Pod/Classes/BONText.m

```diff
@@ -285,6 +285,16 @@ - (NSDictionary *)attributes
         attributes[NSParagraphStyleAttributeName] = paragraphStyle;
     }
 
+    // Text Underlining
+
+    if (self.underlineStyle != NSUnderlineStyleNone) {
+        attributes[NSUnderlineStyleAttributeName] = @(self.underlineStyle);
+    }
+
+    if (self.underlineColor != nil) {
```

Comment:
#pp I typically don't use an explicit `nil` check in this project, preferring instead `if (self.underlineColor) {` for object `nil` checks. But I know that Swift requires an explicit check, which I like, so the only argument against doing it here is consistency with the rest of the code base.